inst/rstudio/templates/project/resources/common/index.Rmd

```diff
@@ -0,0 +1,55 @@
+--- 
+title: "A Minimal Book Example"
+author: "John Doe"
+date: "`r Sys.Date()`"
+site: bookdown::bookdown_site
+url: https://bookdown.org/username/bookdown-demo
+cover-image: your-social-sharing-image.png
```

Comment:
I feel it would be better to have a working example here like:

```suggestion
# Please update the info below according to the actual url
# and cover-image of your book. If you do not have a cover
# image, you may delete url and cover-image below.
url: https://bookdown.org/yihui/bookdown
cover-image: images/cover.jpg
```

otherwise I prefer commenting out these fields:

```yaml
# url: your book url like https://bookdown.org/yihui/bookdown
# cover-image: path to the cover image like images/cover.jpg
```

I'm okay with either way.

Reply:
I preferred the second option here, so I made this change.